R/module_init_data.R

```diff
@@ -126,11 +126,18 @@ srv_init_data <- function(id, data) {
   vapply(
     datanames,
     function(dataname, datasets) {
-      hash <- rlang::hash(data[[dataname]])
+      x <- data[[dataname]]
+
+      code <- if (is.function(x) && !is.primitive(x)) {
+        x <- deparse1(x)
```

Comment:
Ok, so for functions it `deparses` it and pastes into a string of length 1 right? Previously (without `deparse1`) it was possible that `bquote(rlang::hash(.(as.name(dataname))))` had length > 1 ?

I'm trying to understand the need to deparse1 functions

Reply:
Love it, thanks for the explanation!